mage_ai/frontend/components/DataTable/index.tsx

```diff
@@ -249,23 +253,37 @@ function Table({
             >
               {firstColumn && cell.render('Cell')}
               {!firstColumn && (
-                <>
-                  {cellValue === true && 'true'}
-                  {cellValue === false && 'false'}
-                  {(cellValue === null || cellValue === 'null') && 'null'}
-                  {cellValue !== true
+                <FlexContainer justifyContent={'space-between'}>
+                  <Text danger={isInvalid} wordBreak>
+                    {cellValue === true && 'true'}
+                    {cellValue === false && 'false'}
+                    {(cellValue === null || cellValue === 'null') && 'null'}
+                    {cellValue !== true
                     && cellValue !== false
                     && cellValue !== null
                     && cellValue !== 'null'
                     && cellValue
                   }
-                </>
+                  </Text>
+                  {isInvalid && (
+                    <NextLink
+                      as={`/datasets/${slug}/?tabs[]=Reports&show_columns=1&column=${index}`}
+                      href="/datasets/[...slug]"
+                      passHref
+                    >
+                      <Link danger>
+                        View all
+                      </Link>
+                    </NextLink>
+                  )}
+                </FlexContainer>
               )}
             </div>
           );
         })}
       </div>
     );
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@nathaniel-mage Are we sure we shud just ignore the warning instead of updating the dependency array?

Reply:
I was worried about rerender issues, but I'll keep in all the dependencies.